src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1298,14 +1299,21 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
-    void CreateConfigurationProcessor(Context& context)
+    void CreateConfigurationProcessorInternal(Context& context)
     {
         auto progressScope = context.Reporter.BeginAsyncProgress(true);
         progressScope->Callback().SetProgressMessage(Resource::String::ConfigurationInitializing());
 
         anon::ConfigureProcessorForUse(context, ConfigurationProcessor{ anon::CreateConfigurationSetProcessorFactory(context) });
     }
 
+    void CreateConfigurationProcessor(Context& context)
+    {
+        context <<
+            ExecuteUriValidation(UriValidationSource::ConfigurationSource) <<

```

Comment:
Could `ExecuteUriValidation` happen after we set up the progress reporting and message? Just in case it takes a while